lxd/storage/drivers/utils.go

```diff
@@ -826,7 +826,7 @@ func BlockDiskSizeBytes(blockDiskPath string) (int64, error) {
 		defer f.Close()
 		fd := int(f.Fd())
 
-		// Retrieve the block size.
+		// Retrieve the block device size.
 		res, err := unix.IoctlGetInt(fd, unix.BLKGETSIZE64)
```

Comment:
@stgraber @ltrager could the use of BLKGETSIZE64 here also be an issue on IBM Z systems?

Reply:
Nope, per its name, it's 64bit so that will work fine.